src/Storefront/Resources/views/components/Sw/Content/Pagination.js

```diff
@@ -1,8 +1,6 @@
 ({ Shopware, ShopwareComponent } = window);
 
-class Pagination extends ShopwareComponent {
-
-    static selector = 'nav[data-component="Pagination"]';
```

Comment:
Since I don't have the selector anymore, it purely works when the JS is named correctly and the component Twig has the matching data-attribute?

But I could not place a JS component "somewhere" else and use it in two different Twig components then? Because the name and directory must always match? I was just thinking of use cases for more "generic" plugins that could be used on small elements that don't have their own component. But maybe thats a different "util" then or something.

Reply:
I don't think this will be the case when we have a capsulated component system. Otherwise, I would think about a more general utility or even adding it to the global singleton. But we could even make it work with the current system by just extending the import map.